lib/sipPeer.js

```diff
@@ -53,15 +53,21 @@ SipPeer.prototype.delete = function(callback){
 SipPeer.prototype.getTns = function(number, callback){
   var url = this.client.concatAccountPath(SITE_PATH + "/" + this.siteId + "/" + SIP_PEER_PATH + "/" + this.id + "/" + TNS_PATH);
   var field = null;
-  if(arguments.length === 2){
-    url += "/" + number;
-  }
-  else{
+  var query = null;
+  if(arguments.length === 1){
+    query = {
+      page: 1,
+      size: 50000
+    };
     callback = number;
-    url += "?" + "page=1&size=50000" 
+    field = "sipPeerTelephoneNumbers";
+  } else if(typeof number === "string" || typeof number === "number" || number instanceof String){
+    url += "/" + number;
+  } else {
+    query = number; 
     field = "sipPeerTelephoneNumbers";
   }
-  this.client.makeRequest("get", url, function(err, res){
+  this.client.makeRequest("get", url, query, function(err, res){
```

Comment:
Can you add a check to make sure that `query` always has `page` and `size`, and set those if not? That way if someone uses a single different query parameter it'll still work as expected

https://github.com/Bandwidth/node-numbers/blob/master/lib/tn.js#L29 has a similar check

Reply:
Gotcha, took care of it